app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com, source, destination = command.split()
+
+    if com != "mv":
+        return
+
+    path_to_move = destination.split("/")
```

Comment:
Paths are currently concatenated manually using string operations, which can lead to cross-platform compatibility issues. Use `os.path.join` to ensure the code works on all operating systems.